offer-manager-domain/src/main/groovy/life/qbic/business/products/create/CreateProduct.groovy

```diff
@@ -13,13 +16,26 @@ import life.qbic.datamodel.dtos.business.services.Product
  *
  */
 class CreateProduct implements CreateProductInput {
```

Comment:
should implement CopyProductOutput

Reply:
I do not think that it should. The create product should be completely independent of the copy use case.